recipes/settings.py

```diff
@@ -228,6 +228,8 @@
 # path for django_js_reverse to generate the javascript file containing all urls. Only done because the default command (collectstatic_js_reverse) fails to update the manifest
 JS_REVERSE_OUTPUT_PATH = os.path.join(BASE_DIR, "cookbook/static/django_js_reverse")
 
+JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', '')
```

Comment:
Given most users installing to a sub-directory probably want this to match SCRIPT_PATH, would it make sense to turn this to `os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_PATH', ''))`

Reply:
Did you mean SCRIPT_NAME? I have tried it as you said but with SCRIPT_NAME and it works fine.
```suggestion
JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_PATH', ''))

```

Reply:
Did you mean SCRIPT_NAME? I have tried it as you said but with SCRIPT_NAME and it works fine.
```suggestion
JS_REVERSE_SCRIPT_PREFIX = os.getenv('JS_REVERSE_SCRIPT_PREFIX', os.getenv('SCRIPT_NAME', ''))
```